schema_enforcer/config.py

```diff
@@ -40,17 +47,9 @@ class Settings(BaseSettings):  # pylint: disable=too-few-public-methods
     data_file_exclude_filenames: List[str] = [".yamllint.yml", ".travis.yml"]
     data_file_automap: bool = True
 
-    ansible_inventory: Optional[str]
+    ansible_inventory: Optional[str] = None
     schema_mapping: Dict = {}
 
-    class Config:  # pylint: disable=too-few-public-methods
-        """Additional parameters to automatically map environment variable to some settings."""
-
-        fields = {
-            "main_directory": {"env": "jsonschema_directory"},
```

Comment:
where are these take into account now?

Reply:
One of them should have been accounted for by the `env_prefix` in the `model_config`. I had to work on setting the **jsonschema_directory** due to changes in Pydantic v2 so it'll properly work.

I added tests for this as well.